x/migration/keeper/msg_server_claim_morse_supplier.go

```diff
@@ -269,28 +317,58 @@ func (k msgServer) ClaimMorseSupplier(
 		return nil, err
 	}
 
-	claimedSupplierStake := morseClaimableAccount.GetSupplierStake()
-	claimedUnstakedBalance := morseClaimableAccount.GetUnstakedBalance()
+	events := []cosmostypes.Msg{
```

Comment:
1. Please create an empty slice + #PUC why we need it.
2. Create the event separately
3. Append it.